### PR TITLE
Refs [TASK] [34385] Create UI and bind data to new screen

### PR DESCRIPTION
**Pull request by hoangphuc612**
## Related Tickets
- [#Task 34385: Create UI and bind data to new screen] https://edu-redmine.sun-asterisk.vn/issues/34385
## WHAT
- Tạo giao diện màn hình new
- Đẩy dữ liệu lên view
## Evidence (Screenshot or Video)
165827488_195111395448641_4649968896873169332_n

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>

### CODE REVIEW

File: app/src/main/java/com/sun/americanroom/screen/new/NewContract.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.sun.americanroom.screen.new
+
+import com.sun.americanroom.data.model.City
+import com.sun.americanroom.utils.BasePresenter
+
+interface NewContract {
+    interface View : NewContract {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/americanroom/screen/new/NewContract.kt]:
```suggestion
interface NewContract {

    interface View : NewContract {
```

File: app/src/main/java/com/sun/americanroom/screen/new/roomitem/NewRoomContract.kt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.sun.americanroom.screen.new.roomitem
+
+import com.sun.americanroom.data.model.NewRoom
+import com.sun.americanroom.data.model.TopRoom
+import com.sun.americanroom.utils.BasePresenter
+
+interface NewRoomContract {
+    interface View : NewRoomContract {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/americanroom/screen/new/roomitem/NewRoomContract.kt]:
```suggestion
interface NewRoomContract {

    interface View : NewRoomContract {
```

File: app/src/main/java/com/sun/americanroom/screen/new/roomitem/NewRoomContract.kt [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import com.sun.americanroom.data.model.NewRoom
+import com.sun.americanroom.data.model.TopRoom
+import com.sun.americanroom.utils.BasePresenter
+
+interface NewRoomContract {
+
+    interface View : NewRoomContract {
+        fun getNewRoomOnSuccess(newRoom: MutableList<NewRoom>)
+
+        fun onError(exception: Exception?)
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/americanroom/screen/new/roomitem/NewRoomContract.kt]:
```suggestion
        fun getNewRoomOnSuccess(newRoom: MutableList<NewRoom>)
        fun onError(exception: Exception?)
```

File: app/src/main/java/com/sun/americanroom/screen/new/roomitem/adapterroom/ItemNewRoomViewHolder.kt [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        itemView.apply {
+            if (newRoom.rating.toString().equals(Constant.NULL)) {
+                cardViewRating.isVisible = false
+            } else {
+                textViewRating.text = newRoom.rating.toString()
+            }
+            textViewRoomName.text = newRoom.name.toString()
+            Glide.with(context)
+                .load(newRoom.image)
+                .into(imageViewNewRoom)
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/americanroom/screen/new/roomitem/adapterroom/ItemNewRoomViewHolder.kt]: dùng ext

Comment (hoangphuc612) [reply app/src/main/java/com/sun/americanroom/screen/new/roomitem/adapterroom/ItemNewRoomViewHolder.kt]: Dạ vâng anh

**Pull request merged**